modules/deploy/pages/deployment-option/cloud/configure-privatelink-private-service-connect-in-cloud-ui.adoc

```diff
@@ -0,0 +1,88 @@
+= Configure PrivateLink and Private Service Connect in the Cloud UI
+:description: Set up AWS PrivateLink and GCP Private Service Connect in the Redpanda Cloud UI.
+:page-cloud: true
+
+include::shared:partial$feature-flag.adoc[]
+
+NOTE: This guide is for configuring AWS PrivateLink and GCP Private Service Connect using the Redpanda Cloud UI. See xref:deploy:deployment-option/cloud/aws-privatelink.adoc[Configure AWS PrivateLink for Redpanda Cloud] or xref:deploy:deployment-option/cloud/gcp-private-service-connect.adoc[Configure GCP Private Service Connect for BYOC] if you want to set up these services using the API.
+
+The Redpanda AWS PrivateLink and GCP Private Service Connect services provide secure access to Redpanda Cloud from your own VPC. Traffic over PrivateLink and Private Service Connect do not go through the public internet because these connections are treated as their own private AWS and GCP services. While your VPC has access to the Redpanda VPC, Redpanda cannot access your VPC.
+
+Consider using the endpoint services if you have multiple VPCs and could benefit from a more simplified approach to network management:
+
+* PrivateLink and Private Service Connect allow overlapping CIDR ranges in VPC networks.
+* PrivateLink and Private Service Connect do not limit the number of VPC connections.
+* You control which GCP projects or AWS resources are allowed to connect to the endpoint services.
+
+== Requirements
+
+[tabs]
+====
+AWS::
++
+* Your Redpanda cluster and VPC must be in the same region.
+* Use the https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html[AWS CLI] to create a new client VPC or modify an existing one to use the PrivateLink endpoint.
+GCP::
++
+* Use https://cloud.google.com/sdk/docs/install[gcloud^] to create the consumer-side resources, such as a VPC and forwarding rule, or modify an existing one to use the Private Service Connect service attachment created for your cluster.
+====
+
+== Enable endpoint service for existing clusters
```

Comment:
@0x5d @paulzhang97 The Figma designs include a screen for errors in the cluster update to enable the service. Is there anything there we need to document? e.g. any specific errors to expect, how to troubleshoot/fix?

Reply:
Oof. A handful, at least for PSC haha. I need to think about this. For the moment, let's have them contact support.

Reply:
Agree with `contact support`.